src/Apps/Order2/Routes/Checkout/Components/PaymentStep/Order2PaymentForm.tsx

```diff
@@ -99,15 +99,20 @@ const PaymentFormContent: React.FC<PaymentFormContentProps> = ({ order }) => {
   const updateOrderMutation = useUpdateOrderMutation()
   const { setConfirmationToken } = useCheckoutContext()
   const [isSubmittingToStripe, setIsSubmittingToStripe] = useState(false)
+  const [errorMessage, setErrorMessage] = useState<string | null>(null)
+  const [subtitleErrorMessage, setSubtitleErrorMessage] = useState<
+    string | null
+  >(null)
+  const [selectedPaymentMethod, setSelectedPaymentMethod] = useState<
+    "none" | "saved" | "stripe" | "wire"
+  >("none")
+  const defaultErrorMessage =
+    "Something went wrong. Please try again or contact orders@artsy.net"
```

Comment:
non-blocking - probably good to move this out of the component body if it will eventually be re-used for other error messages.